Cargo.toml

```diff
@@ -14,7 +14,7 @@ resolver = "2"
 version = "1.1.1"
 authors = ["thin-edge.io team <info@thin-edge.io>"]
 edition = "2021"
-rust-version = "1.70"
+rust-version = "1.78"
```

Comment:
You also have to update [the version used by CI](https://github.com/thin-edge/thin-edge.io/blob/main/.github/workflows/build-workflow.yml#L62)

Reply:
Sorry, I noticed a bit that this is what you propose with https://github.com/thin-edge/thin-edge.io/pull/2965

Reply:
Yes, I prefer to do MSRV update in separate PR so I will reverse that change of `rust-version` here.

Reply:
Yes, perfect